pkg/clustermgmt/clusterWatch.go

```diff
@@ -166,6 +177,28 @@ func isClusterMissing(err error) bool {
 	return strings.Contains(err.Error(), "could not find the requested resource")
 }
 
+func transformKlusterletAddonConfig(klusterletAddonConfig *agentv1.KlusterletAddonConfig) db.Resource {
+	props := make(map[string]interface{})
+	props["kind"] = "Cluster"
+	props["name"] = klusterletAddonConfig.Spec.ClusterName
+	props["_clusterNamespace"] = klusterletAddonConfig.Spec.ClusterNamespace
+	enabledAddons := map[string]interface{}{}
+	enabledAddons["searchcollectorAddonEnabled"] = klusterletAddonConfig.Spec.SearchCollectorConfig.Enabled
+	enabledAddons["policycontrollerAddonEnabled"] = klusterletAddonConfig.Spec.PolicyController.Enabled
+	enabledAddons["certpolicycontrollerAddonEnabled"] = klusterletAddonConfig.Spec.CertPolicyControllerConfig.Enabled
+	enabledAddons["applicationmanagerAddonEnabled"] = klusterletAddonConfig.Spec.ApplicationManagerConfig.Enabled
+	enabledAddons["iampolicycontrollerAddonEnabled"] = klusterletAddonConfig.Spec.IAMPolicyControllerConfig.Enabled
+	props["addonStatus"] = enabledAddons // maps to the enabled addons on the cluster
```

Comment:
1. Should we match the add-on names from the cluster page (see screenshot below) or the API?
2. I'd like to simplify the key names to make it easier to search.  The search query will be `addon:search-collector=false` vs. `addonStatus:searchCollectorAddonEnabled=false`
3. Missing work-manager addon.

```suggestion
	enabledAddons := map[string]interface{}{}
	enabledAddons["search-collector"] = klusterletAddonConfig.Spec.SearchCollectorConfig.Enabled
	enabledAddons["policy-controller"] = klusterletAddonConfig.Spec.PolicyController.Enabled
	enabledAddons["cert-policy-controller"] = klusterletAddonConfig.Spec.CertPolicyControllerConfig.Enabled
	enabledAddons["application-manager"] = klusterletAddonConfig.Spec.ApplicationManagerConfig.Enabled
	enabledAddons["iam-policy-controller"] = klusterletAddonConfig.Spec.IAMPolicyControllerConfig.Enabled
	props["addon"] = enabledAddons // maps to the enabled addons on the cluster
```

![image](https://user-images.githubusercontent.com/4671325/120387356-cdc6ca80-c2f7-11eb-916c-7f991d038ae8.png)

Reply:
work manager status is not coming from KAC